packages/analytics-core/src/plugins/destination.ts

```diff
@@ -254,10 +269,13 @@ export class Destination implements DestinationPlugin {
     this.config.loggerProvider.warn(getResponseBodyString(res));
 
     this.config.flushQueueSize /= 2;
-    this.addToQueue(...list);
+
+    const tryable = this.tryableList(list);
+    this.scheduleTryable(tryable);
   }
 
   handleRateLimitResponse(res: RateLimitResponse, list: Context[]) {
+    //console.log(list);
```

Comment:
nit: delete it

Reply:
Deleted in https://github.com/amplitude/Amplitude-TypeScript/pull/689/commits/3b954b4c61f9715be3dfff3ed6fb7e9bdc7d4d35